src/Cli/dotnet/commands/dotnet-help/LocalizableStrings.resx

```diff
@@ -180,6 +180,9 @@
   <data name="PackDefinition" xml:space="preserve">
     <value>Create a NuGet package.</value>
   </data>
+  <data name="PackageDefinition" xml:space="preserve">
+    <value>Provides additional package commands.</value>
```

Comment:
I'm not sure if this is the best description for the package. Open to suggestions :)